core/src/main/java/org/apache/calcite/sql/dialect/HiveSqlDialect.java

```diff
@@ -155,6 +155,13 @@ public HiveSqlDialect(Context context) {
   @Override public @Nullable SqlNode getCastSpec(final RelDataType type) {
     if (type instanceof BasicSqlType) {
       switch (type.getSqlTypeName()) {
+      case REAL:
+        return new SqlDataTypeSpec(
+            new SqlAlienSystemTypeNameSpec(
+                "FLOAT",
```

Comment:
Please keep your coding style consistent with the others.

Reply:
Thanks，done @NobiGo